lib/protocol/task-graph-runner.js

```diff
@@ -228,42 +228,30 @@ function taskGraphRunnerProtocolFactory (
         );
     };
 
-    TaskGraphRunnerProtocol.prototype.runTaskGraph = function (name, options, target) {
-        // TODO: BBP what asserts to we need here?
-        return messenger.request(
+    TaskGraphRunnerProtocol.prototype.runTaskGraph = function (graphId, domain) {
+        assert.uuid(graphId);
+        domain = domain || Constants.DefaultTaskDomain;
```

Comment:
Why is this a uuid assert while in other places it is a string assert for graphId?